src/features/selected-post/selected-post-body.tsx

```diff
@@ -0,0 +1,73 @@
+import { styled } from 'styled-components';
+import { useState } from 'react';
+import { PostCardModel } from '#ui/post_card/post-card.model';
+
+import { SelectedPostLikes } from '#features/selected-post-likes/selected-post-likes';
+import { PostNavigationFooter } from '#features/post-navigation-footer/selected-post-navigation-footer';

```

Comment:
if your feature is related to another feature, you should collocate it, not create separated features:

```
/features/selected-post/selected-post-body
/features/selected-post/selected-post-navigation-footer
```